Tools/gdb/libpython.py

```diff
@@ -468,7 +471,7 @@ def __init__(self, cl_name, attrdict, address):
     def __repr__(self):
```

Comment:
Above this at around line 453 (can't select this on GitHub for some reason), there's another `.iteritems()` call that should probably be replaced with `iteritems()`.

Reply:
Here `pyop_attrdict` is not a `dict`, but a `PyDictObjectPtr`:
https://github.com/python/cpython/blob/557efd0f4ffbc5c2274e73c805dbebc5afd1f869/Tools/gdb/libpython.py#L449-L453
Which implements the method:
https://github.com/python/cpython/blob/557efd0f4ffbc5c2274e73c805dbebc5afd1f869/Tools/gdb/libpython.py#L685-L689
I suppose this was done to be consistent with py2.7.

Reply:
Oops sorry I missed that, thank you for the explanation! You can mark this as resolved :), I can't seem to find the button on my end.